package.json

```diff
@@ -33,6 +35,7 @@
     "eslint-config-prettier": "6.15.0",
     "husky": "1.3.1",
     "jest": "28.1.3",
+    "jest-junit": "14.0.1",
```

Comment:
It's odd these came up in this PR. They are dev deps at the top level so it's fine, but I am curious if it was due to being on node v18 that triggered the necessity.

Reply:
I'd be up for trying to remove them now and see if everything still compiles/looks ok